Source/WebInspectorUI/UserInterface/Views/TreeElement.js

```diff
@@ -108,18 +108,6 @@ WI.TreeElement = class TreeElement extends WI.Object
         this.didChange();
     }
 
-    get titleHTML()
-    {
-        return this._titleHTML;
-    }
-
-    set titleHTML(x)
-    {
-        this._titleHTML = x;
-        this._setListItemNodeContent();
-        this.didChange();
-    }
-
```

Comment:
These aren't use anywhere.

Reply:
then we should also remove the references to it in `_setListItemNodeContent` :)